src/Core/DesignTokens/Services/ThemeService.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Drawing;
+using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.Fast.Components.FluentUI.DesignTokens;
+
+internal sealed class ThemeService : IThemeService
```

Comment:
Why the service is not public?

Reply:
Usually, I prefer to have concrete implementation as internal, seems like that applied here. I'll get that changed to public

Reply:
Yes, if the service is used only in the Lib. But in this case is it a Service usable by users/devs? No?

Reply:
I guess, around 99% use cases we don't need to expose the concrete implementation but the other 1% we might have a use-case where the user must be using a different DI container or maybe doing some custom implementation etc..

Anyway, we'll expose the concrete implementation and let the users decide what to do.